### DISCUSSION TITLE
230717 git merge 전략 회의 (13:30 ~ 13:55)

### DISCUSSION THREAD

**Discussion by somsom13** [woowacourse-teams/2023-shook#41] [회의]
## 회의 안건

develop, main 브랜치 merge 전략을 정한다.
- merge
- squash

**Comment by Cyma-s** [comment]
feature 브랜치에서 main으로 머지되는 방식은 Squash & Merge 방식을 채택하였다.
[Github Branch 전략](https://github.com/woowacourse-teams/2023-shook/wiki/Git-Branch-%EC%A0%84%EB%9E%B5) 에 업데이트 되었음.
